### ISSUE TITLE
Textfile: Duplicate filename detected in 0.27.0

### ISSUE THREAD

**Issue by davideverall** [prometheus-community/windows_exporter#1569]
### Current Behavior

After upgrading from 0.25.x or 0.26.0 the textfile collector is reporting duplicate filename where there is no duplicate.

**Error message:**

`ts=2024-08-13T08:52:33.522Z caller=textfile.go:325 level=error collector=textfile msg="Duplicate filename detected: \"C:\\\\Program Files\\\\windows_exporter\\\\textfile_inputs\\\\test_textfile_metric.prom\". Skip File."`

**Verified on:**

- Windows Server 2019 Standard
- windows_exporter v0.27.0

### Expected Behavior

There should be no error as there is no duplicate filename.

### Steps To Reproduce

```Markdown
1. Install windows_exporter version 0.27.0 with the textfile collector enabled
2. Create a textfile in `C:\Program Files\windows_exporter\textfile_inputs` with .prom file extension
3. Check application log for errors
```

### Environment

- windows_exporter Version: 0.27.0
- Windows Server Version: 2019 Standard

### windows_exporter logs

```shell
ts=2024-08-13T08:47:28.935Z caller=exporter.go:229 level=info msg="Enabled collectors: system, process, textfile, scheduled_task, logical_disk, physical_disk, net, service, tcp, printer, cpu, cs, os"

ts=2024-08-13T08:52:33.522Z caller=textfile.go:325 level=error collector=textfile msg="Duplicate filename detected: \"C:\\\\Program Files\\\\windows_exporter\\\\textfile_inputs\\\\test_textfile_metric.prom\". Skip File."

```
```


### Anything else?

_No response_

**Comment by jkroepke** [comment]
Hi, could you please check, if https://github.com/prometheus-community/windows_exporter/actions/runs/10376982647/artifacts/1808559184 solves the issue?

**Comment by jkroepke** [comment]
Let me know, if 0.27.1 does not solve your issue

**Comment by davideverall** [comment]
Thanks @jkroepke 

I can confirm that this is fixed in 0.27.1
